components/host-dashboard/reports-section/HostFeesSection.js

```diff
@@ -1,68 +1,29 @@
-import React, { useMemo, useState } from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { useQuery } from '@apollo/client';
 import { ChevronDown } from '@styled-icons/fa-solid/ChevronDown/ChevronDown';
 import { ChevronUp } from '@styled-icons/fa-solid/ChevronUp/ChevronUp';
-import dynamic from 'next/dynamic';
-import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
-const Chart = dynamic(() => import('react-apexcharts'), { ssr: false });
-
-import { get, groupBy } from 'lodash';
+import { FormattedMessage } from 'react-intl';
 import styled from 'styled-components';
 
 import { CollectiveType } from '../../../lib/collective-sections';
 import { formatCurrency } from '../../../lib/currency-utils';
+import dayjs from '../../../lib/dayjs';
 import { API_V2_CONTEXT, gqlV2 } from '../../../lib/graphql/helpers';
-import { i18nTransactionSettlementStatus } from '../../../lib/i18n/transaction';
 
 import PeriodFilter from '../../budget/filters/PeriodFilter';
 import CollectivePickerAsync from '../../CollectivePickerAsync';
 import Container from '../../Container';
-import ContainerOverlay from '../../ContainerOverlay';
 import { Box, Flex } from '../../Grid';
 import Image from '../../Image';
 import Loading from '../../Loading';
 import StyledLinkButton from '../../StyledLinkButton';
-import { StyledSelectFilter } from '../../StyledSelectFilter';
-import StyledSpinner from '../../StyledSpinner';
 import { P, Span } from '../../Text';
 
-const hostFeeSectionQuery = gqlV2/* GraphQL */ `
-  query HostFeeSection($hostSlug: String!, $dateFrom: DateTime!, $dateTo: DateTime!) {
-    host(slug: $hostSlug) {
-      id
-      legacyId
-      createdAt
-      currency
-      hostMetricsTimeSeries(dateFrom: $dateFrom, dateTo: $dateTo, timeUnit: MONTH) {
```

Comment:
@SudharakaP In this file, there was a confusion between the normal view and the historical one. See this query: we're fetching the historical data but calling it `hostFeeSectionQuery` while the other query is `hostFeeSectionTimeSeriesQuery`.